telemetry-python/src/can_db.py

```diff
@@ -2,27 +2,45 @@
 # this is the code that only interacts with the database
 
 import sqlite3
+import psycopg2
 import pkg_resources, os
+from dataclasses import dataclass
 from pathlib import Path
 from src import ROOT_DIR
 
+# Classes for the different databases
+@dataclass
+class SQLiteEngine:
+    filename: str
```

Comment:
It may be better to store the path to the file instead of the file name. 

```suggestion
    path: Path
```

This would let the caller declare where the db file should be, instead of always placing it in `src/resources` as we do now.